backend/lib/endpoints/schema/users.js

```diff
@@ -78,6 +78,11 @@ const getUserByIdSchema = {
   params: strictSchema().prop("userId", S.string().required()),
 };
 
+const setUserPermissionsSchema = {
+  body: strictSchema().prop("role", S.string().maxLength(100)),
+  params: strictSchema().prop("userId", S.string().required()),
+};
+
```

Comment:
same as above comment for `description` and `response` codes

Reply:
yeah I am not sure if this endpoint should be documented. it is for internal dashboard use.